src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetCalculator.ts

```diff
@@ -397,8 +397,10 @@ export class VexFlowMusicSheetCalculator extends MusicSheetCalculator {
    * @param hasPitchedNote
    */
   protected layoutVoiceEntry(voiceEntry: VoiceEntry, graphicalNotes: GraphicalNote[], graphicalStaffEntry: GraphicalStaffEntry,
-                             hasPitchedNote: boolean): void {
-    return;
+                             hasPitchedNote: boolean, staffIndex: number): void {
+      for (let i: number = 0; i < graphicalNotes.length; i++) {
+        graphicalNotes[i] = MusicSheetCalculator.stafflineNoteCalculator.positionNote(graphicalNotes[i], staffIndex);
```

Comment:
better not extend the function paramters but use: voiceEntry.ParentSourceStaffEntry.ParentStaff.idInMusicSheet to get the staffIndex.